src/renderer/about.jsx

```diff
@@ -26,16 +26,18 @@ const AboutElement = () => (
                 maxHeight: '10rem'
             }}
         /></div>
-        <h2>{productName}</h2>
-        <div>Version {version}</div>
-        <table style={{fontSize: 'x-small'}}>
-            {
-                ['Electron', 'Chrome'].map(component => {
-                    const componentVersion = process.versions[component.toLowerCase()];
-                    return <tr key={component}><td>{component}</td><td>{componentVersion}</td></tr>;
-                })
-            }
-        </table>
+        <div style={{margin: '1.5rem'}}>
```

Comment:
This change is cosmetic: since "Scratch" is so much shorter than "Scratch Desktop" it look odd without some extra margin.